src/components/posts/PostsPage.js

```diff
@@ -44,12 +71,43 @@ const PostsPage = () => {
     navigate("../", { replace: true });
   };
 
+  function createCohort(event) {
+    event.preventDefault();
+    client.post('/cohort').then((res) => {
+     
+      if (res.data.status === 'success') {
+        alert(`cohort ${res.data.data.cohort.id} created`);
```

Comment:
Let's not use `alert`, go with conditionally rendering an element to display a message instead. Browsers can block alerts.